src/replies.py

```diff
@@ -108,6 +111,9 @@ def smiley(n):
 	types.KARMA_NOTIFICATION:
 		em( "You've just been given sweet karma! (check /info to see your karma"+
 			" or /toggleKarma to turn these notifications off)" ),
+	types.DM_REQUEST_NOTIFICATION: lambda username, **_: em( "{username!x} has requested contact in response to this message." + 
+			"\nRun /toggleRequests to turn these notifications off." ),
```

Comment:
this should probably use the same `tg://user` as signing does, not everyone has an username

Reply:
Good point! Forgot that by default tg users don't have them